notebook/exploration-reports/2021.09-dag-pb-encryption.md

```diff
@@ -0,0 +1,202 @@
+# DAG-PB Encryption
+ 
```

Comment:
> or piggy-back existing encryption standards more

I purposely tried to stay very vanilla with the encryption. We are simply encrypting blocks of data. The only interesting bit is that we are doing convergent encryption by using the hash of the object as the key. This has been done before and is well understood. If you are aware of any related standards that would be helpful I'll gladly consider if they provide any value.